views/produto/admin_novo.php

```diff
@@ -0,0 +1,21 @@
+<h3>Novo Produto</h3>
+<form method="POST" action="">
+    <div class="form-group">
+        <label for="titulo">Nome</label>
```

Comment:
os o atributo for das labels estão sempre apontando pra "titulo" e não para os campos certos